internal/host/host.go

```diff
@@ -728,6 +736,27 @@ func (m *Manager) ReportValidationFailedMetrics(ctx context.Context, h *models.H
 	return nil
 }
 
+func (m *Manager) reportValidationChangedMetrics(ctx context.Context, vc *validationContext, h *models.Host, newValidationRes map[string][]validationResult) error {
+	log := logutil.FromContext(ctx, m.log)
+	var currentValidationRes map[string][]validationResult
+	if h.ValidationsInfo != "" {
+		if err := json.Unmarshal([]byte(h.ValidationsInfo), &currentValidationRes); err != nil {
+			log.WithError(err).Errorf("Failed to unmarshal validations info from host %s in cluster %s", h.ID, h.ClusterID)
+			return err
+		}
+		for vCategory, vRes := range newValidationRes {
+			for i, v := range vRes {
+				if v.Status == ValidationFailure && currentValidationRes[vCategory][i].Status == ValidationSuccess {
```

Comment:
Is it only one sided? success -> failure, i don't think that those events are very friendly.
Think of the user that will open the events log and see a lot of failures.

Reply:
> Is it only one sided?
Yes, this is what we agreed on in the meeting we has about it.

> Think of the user that will open the events log and see a lot of failures.
He will get an event for each validation that changed, for example if one of hist host got disconnected, he will get an event for it.

This was agreed by @ErezAlster and @avishayt that those events are important for the host lifecycle, and there is no reason to have a metrics for it and don't have an event as well.

Reply:
@filanov will it actually be a lot?

Reply:
Even if it's not a lot, users will just see failures do you think that it's good?

Reply:
Don't we want to let the user know that his host doesn't have enough CPU for example?
Would it be better to just not letting him install without he knowing why?

IMO those events are informative.

Reply:
I mean that it will be nice to let the user know that validations changed to success, it will help in the full flow and to debug.

Reply:
Added second event when the validation got fixed.